modules/imgproc/src/color_yuv.simd.hpp

```diff
@@ -1395,9 +1397,10 @@ struct YUV420p2RGB8Invoker : ParallelLoopBody
 #define MIN_SIZE_FOR_PARALLEL_YUV420_CONVERSION (320*240)
 
 template<int bIdx, int uIdx, int dcn>
-inline void cvtYUV420sp2RGB(uchar * dst_data, size_t dst_step, int dst_width, int dst_height, size_t _stride, const uchar* _y1, const uchar* _uv)
+inline void cvtYUV420sp2RGB(uchar * dst_data, size_t dst_step, int dst_width, int dst_height,
+                            const uchar* _y1, const uchar* _uv, size_t _y1_step, size_t _uv_step)
 {
-    YUV420sp2RGB8Invoker<bIdx, uIdx, dcn> converter(dst_data, dst_step, dst_width, _stride, _y1,  _uv);
+    YUV420sp2RGB8Invoker<bIdx, uIdx, dcn> converter(dst_data, dst_step, dst_width, _y1,  _uv, _y1_step, _uv_step);
     if (dst_width * dst_height >= MIN_SIZE_FOR_PARALLEL_YUV420_CONVERSION)
         parallel_for_(Range(0, dst_height/2), converter);
```

Comment:
Is it safe for images with odd heights?

Reply:
`Y` plane must have even width and height values (see 420 format).

Reply:
Seems this is required by the format definition.

Reply:
> `Y` plane must have even width and height values (see 420 format).

Noted with thanks :)